docs/components/modules/ROOT/pages/irc-component.adoc

```diff
@@ -29,7 +29,6 @@ for this component:
 
 [source,java]
 ---------------------------------------------------------------------
-irc:nick@host[:port]/#room[?options]
 irc:nick@host[:port]?channels=#channel1,#channel2,#channel3[?options]
```

Comment:
this seems not correct, in toher places, it is mentioned `irc:nick@host[:port]?[options]`